### PR TITLE
Added missing options and updated methods section for menu widget

### PR DESCRIPTION
**Pull request by vasilii-b**
This pull request (PR) aims to enrich the documentation for Menu Widget (#1276) with the following:
- Add missing widget options
- Add missing widget methods

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_menu.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_menu.html
- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_menu.html

## Links to Magento source code

**Magento 2.1**
- https://github.com/magento/magento2/blob/2.1/lib/web/mage/menu.js#L23
- https://github.com/magento/magento2/blob/2.1/lib/web/mage/menu.js#L96
- https://github.com/magento/magento2/blob/2.1/lib/web/mage/menu.js#L218

**Magento 2.2**
- https://github.com/magento/magento2/blob/2.2/lib/web/mage/menu.js#L22-L23
- https://github.com/magento/magento2/blob/2.2/lib/web/mage/menu.js#L648
- https://github.com/magento/magento2/blob/2.2/lib/web/mage/menu.js#L619

**Magento 2.3**
- https://github.com/magento/magento2/blob/2.3/lib/web/mage/menu.js#L22-L23
- https://github.com/magento/magento2/blob/2.3/lib/web/mage/menu.js#L646
- https://github.com/magento/magento2/blob/2.3/lib/web/mage/menu.js#L617

whatsnew
Added missing [menu widget](https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_menu.html) options and methods.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by vasilii-b** [comment]
Hi @ronak2ram ,
Requested changes were added. Please proceed. Thanks!

File: guides/v2.1/javascript-dev-guide/widgets/widget_menu.md [hunk 1]
```diff
@@ -40,6 +41,14 @@ Setting the default responsive handler for the navigation widget.
 
 **Default value**: `false`
 
+### `mediaBreakpoint` {#m_mediaBreakpoint}
+Sets the breakpoint where menu changes approach on how menu is shown and reacts
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_menu.md]: This needs a little clarification. Consider the following rewrite:

> Sets the width of user’s window in pixels for which the menu switches between mobile view and desktop view.

or

> Set of options that can be dynamically set while the page is resizing.

Don't forget to make the same changes to the 2.3 file.

**Comment by vasilii-b** [comment]
Hi @jeff-matthews ,
Please consider added changes as requested. Thanks!

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**